benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +174,24 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    pretty_name = "Fit Methods Overriden with Analytical MLEs"
```

Comment:
Overridden

Reply:
```suggestion
    pretty_name = "Fit Methods Overridden with Analytical MLEs"
```

Reply:
I'm not sure if we just want to benchmark the ones that have overridden fit methods because then there will be no change. I think you want to benchmark all of the distribution fit methods so you can see the drop when the overridden fit method is merged.

Reply:
Summing up the combinations for fitting a distribution seems preferable over trying to write (many) methods to run to account for each combination of fixed parameters.